apps/lib/blockly/ar_sa.js

```diff
@@ -36,7 +36,6 @@ Blockly.Msg.CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE = "تخطي ما تبقى
 Blockly.Msg.CONTROLS_FLOW_STATEMENTS_WARNING = "تحذير: يمكن استخدام هذا المربع البرمجي فقط داخل حلقة.";
 Blockly.Msg.CONTROLS_FOREACH_HELPURL = "https://code.google.com/p/blockly/wiki/Loops#for_each\n";
 Blockly.Msg.CONTROLS_FOREACH_INPUT_INLIST = "في قائمة";
-Blockly.Msg.CONTROLS_FOREACH_INPUT_INLIST_TAIL = "";
```

Comment:
Any idea why we're not syncing all of the no-translation strings this week instead of last week?

Reply:
None at all!